js/plates-model/plate.js

```diff
@@ -252,8 +252,8 @@ export default class Plate extends PlateBase {
       })
     }
 
-    // Old island field is turned into regular ocean.
-    island.isOcean = true
+    // Remove the old island field.
+    island.alive = false
```

Comment:
To be honest I don't *totally* understand the new plate-breaking behavior, but it's interesting that it only took such a small change!